api/src/test/java/org/openmrs/module/datafilter/impl/EncTypePrivilegeBasedFilterTest.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.datafilter.impl;
+
+import org.hibernate.SessionFactory;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.EncounterType;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.datafilter.TestConstants;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
```

Comment:
we can remove the unwanted spaces here

Reply:
Hi @HerbertYiga 

Yes, I just did it

Reply:
Hi @HerbertYiga, @dkayiwa 

Do you feel there is still more changes needed?